eslint.config.mjs

```diff
@@ -0,0 +1,259 @@
+// @ts-check
+import eslint from "@eslint/js";
+import noNull from "eslint-plugin-no-null";
+import simpleImportSort from "eslint-plugin-simple-import-sort";

```

Comment:
If I get it right, `eslint-plugin-no-null` and `eslint-plugin-simple-import-sort` do not ship typings. Unfortunately `// @ts-check` is another painful aspect of flat config. Type checks are broken. This also means that using  `typescript-eslint` with `ProjectService` makes linting `eslint.config.js` very troublesome.

I was trying to explain these problems in ESLint repo, but got ignore. Unfortunately support for JSON flat config got overlooked. They say it's not possible. Dear TypeScript, thanks for doing the impossible (;

Reply:
Oh, right, the default inferred project options have `noImplicitAny=false`.

Reply:
All plugins are now gone.